app/apps/server/bridges/scheduler.js

```diff
@@ -89,4 +170,11 @@ export class AppSchedulerBridge {
 			this.isConnected = true;
 		}
 	}
+
+	listenToSchedulerEvents() {
+		this.scheduler.on('start', (job) => this.orch.debugLog(`Job ${ job.attrs.name } started`));
+		this.scheduler.on('complete', (job) => this.orch.debugLog(`Job ${ job.attrs.name } finished`));
+		this.scheduler.on('success', (job) => this.orch.debugLog(`Job ${ job.attrs.name } was successful`));
+		this.scheduler.on('fail', (job, err) => this.orch.getRocketChatLogger().error(`Job ${ job.attrs.name } has failed: ${ err.message } `));
```

Comment:
I'm not sure about this 🤔  We've had complaints about the Apps-Engine polluting the server's log too much, and people asking to disable it (which is currently unavailable). I'd say we should keep this change for when we add the possibility of configuring the log levels of the Apps-Engine independently.